tools/crm_node.c

```diff
@@ -562,10 +561,11 @@ main(int argc, char **argv)
     }
 
     if (options.dangerous_cmd && options.force_flag == FALSE) {
-        fprintf(stderr, "The supplied command is considered dangerous."
-                "  To prevent accidental destruction of the cluster,"
-                " the --force flag is required in order to proceed.\n");
         exit_code = CRM_EX_USAGE;
+        g_set_error(&error, PCMK__EXITC_ERROR, exit_code,
+                    "The supplied command is considered dangerous."
+                    "  To prevent accidental destruction of the cluster,"
+                    " the --force flag is required in order to proceed.");
```

Comment:
Why convert just this and a few other messages to GError? GError obviously makes sense when used with the glib functions that require it, and it makes some sense when we might want to fprintf() or out->err() conditionally, but I don't see any advantage over just fprintf() in other cases. If we did want to use it more, I'd think we want to convert all of the error messages to it.

Reply:
I am constantly struggling with how much should use GError, so I don't have a good answer on this.  I can put this one back as fprintf, at least.  The GError usage around pcmk__output_new was mainly because some tools were already doing so.  For g_option_context_parse_strv, command line processing is already using GError so it fits better here.

I've additionally considered converting pcmk__output_new to take a GError to save having to do that block everywhere, but it doesn't seem like it saves much.